packages/plugin-pre/src/Pre.ts

```diff
@@ -0,0 +1,50 @@
+import { JWPlugin, JWPluginConfig } from '../../core/src/JWPlugin';
+import { Loadables } from '../../core/src/JWEditor';
+import { Parser } from '../../plugin-parser/src/Parser';
+import { Renderer } from '../../plugin-renderer/src/Renderer';
+import { PreDomParser } from './PreDomParser';
+import { PreDomRenderer } from './PreDomRenderer';
+import { CommandParams } from '../../core/src/Dispatcher';
+import { VNode, isLeaf } from '../../core/src/VNodes/VNode';
+import { distinct } from '../../utils/src/utils';
+import { PreNode } from './PreNode';
+
+export class Pre<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T> {
+    commands = {
+        applyPre: {
```

Comment:
I'm not convinced by the name of this command but to be fair I don't have a better idea yet. Though if we used `applyHeading` for Headings then it makes sense I guess. I think to remember we used `applyHeadingStyle` so I suggest `applyPreStyle` ? Maybe that's a tad too much ? 🐸

Reply:
Let's do that then.